susemanager-utils/cluster-providers/caasp/formulas/caasp-management-node-formula/caasp-management-node-formula.spec

```diff
@@ -0,0 +1,54 @@
+#
+# spec file for package caasp-management-node-formula
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%define fname caasp-management-node
+%define fdir  %{_datadir}/salt-formulas
+Name:           caasp-management-node-formula
+Version:        0.1
```

Comment:
If we don't wan't this at the formulas repository: Is there any good reason for this to be 0.1? We better have all packages with the same `X.Y.Z` versioning. In this case `4.1.0`.